### ISSUE TITLE
Custom Media Selector

### ISSUE THREAD

**Issue by urukai** [bueltge/multisite-global-media#28]
Hello
Your plugin seems to be very promising. When using with a custom button, the left sidebar does not appear. Imagine custom records in a custom table, the administrator can edit some data and add images to the custom records. Then he or she should be able to select images from other blogs. Would this be possible?

**Comment by bueltge** [comment]
It is not possible to add the sidebar or any query to the 'custom button', your function for selecting images?

**Comment by urukai** [comment]
Let me explain the context:

I created a custom table in the database, let's say for my cool projects. So in the dashboard, I added a admin page for displaying a list of all projects saved in this table, with links to edit and delete a project.

When editing a project, I would like to click the said button "Add Slide" (oder ähnlich) and add a image from the main blog. 

<img width="602" alt="screenshot 15" src="https://user-images.githubusercontent.com/5707163/44780705-d4dab000-ab82-11e8-8d2a-e6a68820e85d.png">


But when adding a function on click like


    var m = wp.media({
        title: 'Bild für die Slidewhow wählen',
	library: {
	    type: 'image'
	},
	multiple: false,
	button: {
	    text: 'Auswählen'
	}
	});
	m.open();

then the left sidebar in the media selector is not visible, neither the global tab which is generated by your plugin (see screenshot)
<img width="1146" alt="screenshot 14" src="https://user-images.githubusercontent.com/5707163/44780627-abba1f80-ab82-11e8-970d-8de62b13a294.png">

And this is my question: is it possible to add this global tab in this specific case?
